### PR TITLE
L2D6A1

### PR DESCRIPTION
**Pull request by evyryan**

### CODE REVIEW

File: kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml [hunk 1]
```diff
@@ -68,13 +68,14 @@
...
                <head xml:id="kzz7yh-f19041-Hd1e120">Quaestio 1</head>
+               <head type="question-title">De dignitate a qua ceciderunt</head>
                <p xml:id="kzz7yh-f19041-d1e122">
       ¶ Quaestio I. 
       <lb ed="#V" n="106"/>PRimo ostendo quod lucifer de 
-      fu<lb ed="#V" n="107" break="no"/>permo ordine non 
-      <lb ed="#V" n="108"/>fuerit. Dicit enim Damascen. lib. 3. cap. 4. Quod 
-      <lb ed="#V" n="109"/>lucifer erat de illis angelis qui preerant 
-      terrestri<lb ed="#V" n="110" break="no"/>ordini. Ergo non erat de ordine superiori.
+      fu<lb ed="#V" n="107" break="no"/>permo <!--TODO: abbreviation?--> ordine non
```
Comment (RobbieBuhite) [inline kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml:75]:
```suggestion
      <lb ed="#V" n="107"/>primo ordine non 
```

File: kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml [hunk 2]
```diff
@@ -107,7 +108,7 @@
    </p>
                <p xml:id="kzz7yh-f19041-d1e185">
                   <lb ed="#V" n="5"/>Respondeo quod lucifer inter omnes angelos fuit 
-      <lb ed="#V" n="6"/>creatus nobilior. Et si stetisset: et ad 
+      <lb ed="#V" n="6"/>creatus nobilior. Et si stetisset<!--TODO Abbreviation-->: et ad
```
Comment (RobbieBuhite) [inline kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml:111]:
```suggestion
      <lb ed="#V" n="6"/>creatus nobilior. Et si stetisset: et ad 
```

Comment (RobbieBuhite) [reply kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml:111]: I don't think there is an abbreviation after "Stetisset," I think it's just a colon and then the abbreviaiton for "et."

File: kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml [hunk 3]
```diff
@@ -167,11 +168,12 @@
             </div>
             <div xml:id="kzz7yh-f19041-Dd1e299">
                <head xml:id="kzz7yh-f19041-Hd1e301">Quaestio 2</head>
+               <head type="question-title">Utru de quolibet ordine aliqua multitudo ceciderit</head>
```
Comment (RobbieBuhite) [inline kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml:171]:
```suggestion
               <head type="question-title">Utrum de quolibet ordine aliqua multitudo ceciderit</head>
```

File: kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml [hunk 4]
```diff
@@ -167,11 +168,12 @@
...
                <p xml:id="kzz7yh-f19041-d1e303">
       ¶ Quaestio II. 
       <lb ed="#V" n="49"/>SEcundo queritur vtrum de quolibet ordine 
       <lb ed="#V" n="50"/>aliqua multitudo ceciderit. Et 
-      <lb ed="#V" n="51"/>vetur quod non. Apostolus ad Eph. vltmo dicit. quod non est 
+      <lb ed="#V" n="51"/>vetur quod non. Apostolus ad Eph. vltmo.dicit. quod non est
```
Comment (RobbieBuhite) [inline kzz7yh-f19041/cod-ve09v2_kzz7yh-f19041.xml:176]:
```suggestion
      <lb ed="#V" n="51"/>videtur quod non. Apostolus ad Eph. vltmo.dicit. quod non est 
```

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**